src/core/ext/transport/chttp2/transport/hpack_parser_table.h

```diff
@@ -109,10 +123,11 @@ class HPackTable {
     Memento PopOne();
 
     // Lookup the entry at index, or return nullptr if none exists.
-    const Memento* Lookup(uint32_t index) const;
+    const Memento* Lookup(uint32_t index);
+    const Memento* Peek(uint32_t index) const;
```

Comment:
can you add a comment to tell the difference between the two?

Reply:
done